modules/mobile/spec/request/check_in_demographics_request_spec.rb

```diff
@@ -0,0 +1,99 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_relative '../support/helpers/iam_session_helper'
+
+RSpec.describe 'check in demographics', type: :request do
+  before do
+    allow_any_instance_of(User).to receive(:vha_facility_hash).and_return({
+                                                                            '516' => ['12345'],
+                                                                            '553' => ['2'],
+                                                                            '200HD' => ['12345'],
+                                                                            '200IP' => ['TKIP123456'],
+                                                                            '200MHV' => ['123456']
+                                                                          })
+    iam_sign_in
+  end
+
+  describe 'GET /mobile/v0/appointments/check-in/demographics' do
+    context 'test' do
+      it 'returns expected check in demographics data' do
+        VCR.use_cassette('mobile/check_in/token_200') do
+          VCR.use_cassette('mobile/check_in/get_demographics_200') do
+            get '/mobile/v0/appointments/check-in/demographics', headers: iam_headers,
+                                                                 params: { 'location_id' => '516' }
+          end
+        end
+        expect(response).to have_http_status(:ok)
+        expect(response.parsed_body).to eq(
```

Comment:
while this is a lot, since I didn't think it was worth having a model for this. checking the exact output will be good for calling out any changes we make that may effect the final output in ways we don't expect